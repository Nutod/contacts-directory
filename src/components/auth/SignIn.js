import React from "react";
import Background from "../../components/elements/Background/Background";
import Eclipse from "../../assets/icons/eclipse-bg.svg";
import Logo from "../../components/elements/Logo/Logo";
import Navigation from "../../components/elements/Navigation/Navigation";
import { Form } from "../../components/elements/Form/Form";
import { Button } from "../../components/elements/Button/Button";
import theme from "../../components/elements/Icons/Colors";
import {
	Container,
	HeadingOne,
	ToolBar,
	Content,
	Backdrop,
	TermsAndAgreementParagraph,
	LoginParagraph,
	BrandText,
	HR,
	CTAText,
	CTAWrapper
} from "./SignUp";

export default function SignUp() {
	return (
		<Container>
			<ToolBar>
				<Logo />
				<Navigation />
			</ToolBar>
			<Backdrop>
				<Background />
			</Backdrop>
			<Backdrop>
				<img
					src={Eclipse}
					alt="bg"
					style={{
						width: "140rem",
						position: "absolute",
						top: "0",
						left: "-50rem"
					}}
				/>
			</Backdrop>
			<Content>
				<Form onSubmit={event => event.preventDefault()}>
					<label htmlFor="email">Email</label>
					<input type="email" />
					<label htmlFor="password">Password</label>
					<input type="password" />

					<div style={{ textAlign: "center" }}>
						<TermsAndAgreementParagraph>
							Forgot your Password?{" "}
							<span style={{ color: `${theme.brand}` }}>Reset it here</span>
						</TermsAndAgreementParagraph>
						<Button>Login</Button>
						<LoginParagraph>
							Don't have an accout?{" "}
							<span style={{ color: `${theme.brand}` }}>Get Started</span>
						</LoginParagraph>
					</div>
				</Form>
				<CTAWrapper>
					<HeadingOne>Be More.</HeadingOne>
					<CTAText>
						All your contacts from all your accounts{" "}
						<BrandText>In One Place</BrandText>
					</CTAText>
					<HR />
				</CTAWrapper>
			</Content>
		</Container>
	);
}
