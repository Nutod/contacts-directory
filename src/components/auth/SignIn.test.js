import React from "react";
import Enzyme, { shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import toJson from "enzyme-to-json";
import SignIn from './SignIn'

Enzyme.configure({ adapter: new Adapter() });

describe("<SignIn />", () => {
  const wrapper = shallow(<SignIn />);

  it("Should match snapshot", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
