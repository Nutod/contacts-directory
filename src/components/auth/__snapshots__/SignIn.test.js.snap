// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SignIn /> Should match snapshot 1`] = `
<styled.div>
  <styled.div>
    <Logo />
    <Navigation />
  </styled.div>
  <styled.div>
    <Background />
  </styled.div>
  <styled.div>
    <img
      alt="bg"
      src="eclipse-bg.svg"
      style={
        Object {
          "left": "-50rem",
          "position": "absolute",
          "top": "0",
          "width": "140rem",
        }
      }
    />
  </styled.div>
  <styled.div>
    <styled.form
      onSubmit={[Function]}
    >
      <label
        htmlFor="email"
      >
        Email
      </label>
      <input
        name="email"
        onChange={[Function]}
        required={true}
        type="email"
      />
      <label
        htmlFor="password"
      >
        Password
      </label>
      <input
        name="password"
        onChange={[Function]}
        required={true}
        type="password"
      />
      <div
        style={
          Object {
            "textAlign": "center",
          }
        }
      >
        <styled.p>
          Forgot your Password?
           
          <span
            style={
              Object {
                "color": "#e4235d",
              }
            }
          >
            Reset it here
          </span>
        </styled.p>
        <styled.button>
          Login
        </styled.button>
        <styled.p>
          Don't have an accout?
           
          <span
            style={
              Object {
                "color": "#e4235d",
              }
            }
          >
            Get Started
          </span>
        </styled.p>
      </div>
    </styled.form>
    <styled.div>
      <styled.h1>
        Be More.
      </styled.h1>
      <styled.p>
        All your contacts from all your accounts
         
        <styled.span>
          In One Place
        </styled.span>
      </styled.p>
      <styled.hr />
    </styled.div>
  </styled.div>
</styled.div>
`;
